infrastructure/ecr.tf

```diff
@@ -0,0 +1,7 @@
+resource "aws_ecr_repository" "fake_sierra" {
+  name = "uk.ac.wellcome/fake_sierra"
```

Comment:
```suggestion
  name = "weco/fake_sierra"
```

?

Reply:
Annoyingly, I started with weco and had to change it back because [weco-deploy doesn't like it](https://github.com/wellcomecollection/weco-deploy/pull/101)

Reply:
😭